Document `tfe-admin list-nodes` command (#1938)
hashicorp/terraform-website: content/source/docs/enterprise/admin/active-active.html.md

}
```

#### list-nodes

```bash
tfe-admin list-nodes
```

> This command will list the IP addresses of all active nodes in the active/active installation. Nodes send a heartbeat
> every 5 seconds to signal that they are active. If a heartbeat is not received within 30 seconds, the node is considered
> inactive and removed from this list.

Reviewer: ```suggestion
This command lists the IP addresses of all active nodes in the active/active installation. Nodes send a heartbeat every 5 seconds to signal that they are active. If Terraform does not receive a heartbeat within 30 seconds, it considers the node inactive and removes the node from this list.
```
